src/tests/unit/sync/syncimages_status.f90

```diff
@@ -10,7 +10,6 @@ program sync_images_stat
   me = this_image()
 
   if (me /= 1 ) then
-     call sleep(1)
```

Comment:
So if I'm understanding correctly, all the other images will wait for a response from image 1, but image 1 isn't included in the call to `sync images()` they would wait forever. However, image 1 will reach program termination, and then the other images waiting via `sync images()` will receive a STAT_STOPPED_IMAGE as the response. This is why there is no longer a race condition and/or hang in this test: all the images will be waiting for each other except image 1 which will continue until it terminates, at which point the other images will get the STAT_STOPPED_IMAGE notification. Is my understanding correct?